File: Source/Rio.Models/DataTransferObjects/Account/AccountDto.cs
using System;
using Rio.Models.DataTransferObjects.User;
using System.Collections.Generic;

namespace Rio.Models.DataTransferObjects.Account
{
    public class AccountDto
    {
        public int AccountID { get; set; }
        public int? AccountNumber { get; set; }
        public string AccountName { get; set; }
        public string Notes { get; set; }
        public string AccountVerificationKey { get; set; }
        public DateTime? AccountVerificationKeyLastUseDate { get; set; }
        public List<UserSimpleDto> Users { get; set; }
        public AccountStatusDto AccountStatus { get; set; }
        public string AccountDisplayName { get; set; }
        public string ShortAccountDisplayName { get; set; }
        public int NumberOfParcels { get; set; }
        public int NumberOfUsers { get; set; }
    }
}

File: Source/Rio.Web/src/app/shared/models/account/account-dto.ts
import { UserSimpleDto } from '../user/user-simple-dto';
import { AccountStatusDto } from './account-status-dto';
export class AccountDto {
    AccountID: number;
    AccountName: string;
    AccountNumber: number;
    Notes: string;
    AccountStatus: AccountStatusDto;
    Users: Array<UserSimpleDto>;
    AccountDisplayName: string;
    NumberOfParcels: number;
    NumberOfUsers: number;
    ShortAccountDisplayName: string;
    
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

